context-linux/src/etc/one-context.d/net-99-report-ready

```diff
@@ -69,14 +69,29 @@ while [ "$RETRY_COUNT" -gt 0 ] ; do
             fi
             ;;
     esac
+}
+
+while [ "$RETRY_COUNT" -gt 0 ] ; do
+    if [[ -n "$READY_SCRIPT" ]];then
+        if eval "$READY_SCRIPT";then
```

Comment:
Have you considered for the script to be encoded in base64? You could decode it just before the loop. I guess in general that should be some better practice. :thinking: